api/build.gradle

```diff
@@ -627,6 +627,16 @@ task fetchFireCloudUserProfile(type: JavaExec) {
   }
 }
 
+// See project.rb command: fetch-workspace-details
+task fetchWorkspaceDetails(type: JavaExec) {
+  classpath = sourceSets.tools.runtimeClasspath
+  main = "org.pmiops.workbench.tools.FetchWorkspaceDetails"
+  systemProperties = commandLineSpringProperties
+  if (project.hasProperty("appArgs")) {
```

Comment:
I'm surprised you have to do this for every task that takes arguments. Also, why can't the `Eval.me()` step handle a missing property so you don't need a conditional here?